pkgs/development/python-modules/scikit-survival/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchPypi
+, git
+, setuptools_scm
+, cython
+, scikitlearn
+, cvxpy
+, cvxopt
+, numexpr
+, osqp
+, pandas
+, scipy
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "scikit-survival";
+  version = "0.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0q47ifqrjyviszxz68hcgk10wcajnp9cinvl45h5zw6a478y7j15";
+  };
+
+  # Works with pandas >= 1.0.1
+  # waiting for upstream to merge and release new version
```

Comment:
Is there a PR/version number that you can refer to here for when this patch won't be needed?

Reply:
Yes, will add it.